_projects/2024/100540594/100540594.Rmd

```diff
@@ -0,0 +1,1320 @@
+            color = "white",
+            fontface = "bold",
+            # we need to adjust the lineheight argument for "Korea, \nRep." and 
+            # "Saudi \n Arabia" to make sure that the vertical spacing is fine
+            lineheight = .9) +
+  geom_text(data = country_labels_adj,
+            aes(x = long + 0.1, y = lat, label = Label),
+            size = 1.75,  
+            color = "white",
+            fontface = "bold",
+            lineheight = .9) +
+  geom_text(data = country_labels_adj,
+            aes(x = long, y = lat - 0.1, label = Label),
+            size = 1.75,  
+            color = "white",
+            fontface = "bold",
+            lineheight = .9) +
+  geom_text(data = country_labels_adj,
+            aes(x = long, y = lat + 0.1, label = Label),
+            size = 1.75,  
+            color = "white",
+            fontface = "bold",
+            lineheight = .9) +
+  
+  # add country labels (white)
+  geom_text(data = country_labels_adj,
+            aes(x = long, y = lat, label = Label),
+            size = 1.77,  # slightly larger than the black text
+            color = "white",
+            fontface = "bold",
+            lineheight = .9) +
+  
+  # add country labels (black)
+  geom_text(data = country_labels_adj,
+            aes(x = long, y = lat, label = Label),
+            size = 1.65,  
+            color = "black",
+            fontface = "bold",
+            lineheight = .9) + 
+  
+  # add rectangles for nes areas
+  geom_rect(data = data.frame(
+    xmin = c(special_areas_coords[1,2] - 11.75, 
+             special_areas_coords[2,2] - 22.5),
+    xmax = c(special_areas_coords[1,2]  + 11.75, 
+             special_areas_coords[2,2] + 22.5),
+    ymin = c(special_areas_coords[1,3] - 2.6, 
+             special_areas_coords[2,3] - 2.6),
+    ymax = c(special_areas_coords[1,3] + 2.6, 
+             special_areas_coords[2,3] + 2.6),
+    label = c("Areas, nes", "Rest of America, nes")),
+    aes(xmin = xmin, xmax = xmax, ymin = ymin, ymax = ymax),
+    fill = "#ffffff",
+    alpha = .9,
+    color = "#cccccc", 
+    size = 0.3) +
+  
+  # add text labels for nes areas
+  geom_text(data = data.frame(
+    x = c(special_areas_coords[1,2], special_areas_coords[2,2]),
+    y = c(special_areas_coords[1,3], special_areas_coords[2,3]),
+    label = c("Areas, nes", "Rest of America, nes")),
+    aes(x = x, y = y, label = label),
+    size = 1.65, 
+    fontface = "bold")  + 
+  
+  theme(text = element_text(family = "gudea"))
+
+print(p)
+```
+
+#### Annotations
+
+Finally, we can add the legend 'Scale' in the bottom-left. This legend helps 
+reader understand the trade flows more easily by showing how the thickness of 
+the flows corresponds to trade volume. We leave out the other annotations 
+as these do not add anything meaningful to the plot. 
+
+```{r}
+p <- p + 
+  annotation_custom(
+    grid::rasterGrob(
+      png::readPNG(here("_projects", "2024", "100540594", "legend.png")),
```

Comment:
Same, just the name of the file, without `here("projects", "2024", ...)` please.